Automattic/wp-calypso
# Domains: Add DNSSEC management card

## Proposed Changes

This PR adds a DNSSEC management card to the domain management page.

For this first iteration, users will be able to toggle DNSSEC on and off. Also, when DNSSEC is enabled, they'll be able to view the `DNSKEY` and `DS` record values for their domain. This feature is currently only available for apex domains registered with WPCOM.

### Screenshots

- Placement of the DNSSEC card in the domain management page

![Markup on 2024-07-11 at 16:00:30](https://github.com/user-attachments/assets/7ca9f211-e423-4b74-8652-c3541fa17012)

- DNSSEC disabled

<img width="706" alt="Screenshot 2024-07-11 at 16 01 02" src="https://github.com/user-attachments/assets/1c2ad7cb-4df7-4afb-8d9e-b6a29fd3d87b">

- DNSSEC enabled

<img width="701" alt="Screenshot 2024-07-16 at 18 43 11" src="https://github.com/user-attachments/assets/6e1239f6-1c5c-4b3a-9622-c76096cb4f9f">

## Why are these changes being made?

This is a long-standing user request that we are now able to handle (pMz3w-ich-p2).

## Testing Instructions

- This PR depends on D154951-code
- Open the live Calypso link or build this branch locally
- Go to the domain management page of a domain that was bought in WPCOM
- Ensure the DNSSEC card is present
- Try toggling DNSSEC on and off and ensure no error happens

## Pre-merge Checklist
